cronograma/r-dev-day/index.qmd

```diff
@@ -21,10 +21,10 @@ knitr:
 
 R Dev Day \@ LatinR 2024 es un evento colaborativo donde las personas trabajarán en pequeños grupos para hacer contribuciones a R base, o a la infraestructura que soporta dicha contribución. Ejemplos de posibles contribuciones:
 
--   Traducir mensajes de error cadenasal español/portugués de Brasil
+-   Traducir mensajes de error a español o portugués
```

Comment:
I think is good to let "portugués de Brasil" because we have different types of Portuguese (ex. from Europe/Portugal, from Latam/Brasil, from Africa/Moçambique, etc), and we are focusing in the Brazilian Portuguese

Reply:
sorry about that Bea, I thought the specification for português brasileiro was an artifact of the machine translation

Reply:
No problem, Luis! Thank you for the review!